tools/debian/control

```diff
@@ -77,7 +77,6 @@ Package: cockpit-docker
 Architecture: all
 Depends: ${misc:Depends},
          docker.io (>= 1.3.0) | docker-engine (>= 1.3.0),
-         python,
```

Comment:
Storage.jsx is built into docker.js by webpack. and I believe that is distributed on debian. However, it is true, there is no such thing as debian atomic and it only used for dealing with storage on atomic. So it's probably ok to leave out. My only concern would be that future devs will see use python in some other way in the docker package and forget to update the requirements because technically it was already in use in the docker code. But i'll leave it up to you.